bigml/anomaly.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
-#!/usr/bin/env python
+# !/usr/bin/env python
```

Comment:
in general, i think these lines are not needed and we could remove them all.  but with this change, the line becomes a pure comment, and it's even less useful :)